src/components/impact/SDGBlock.js

```diff
@@ -0,0 +1,241 @@
+    color: '#00689D',
+  },
+  {
+    title: 'Partmerships\nFor The Goals',
+    image: img17,
+    // eslint-disable-next-line max-len
+    textOnOver: 'Strengthen the means of implementation and revitalize the global partnership for sustainable development',
+    color: '#19486A',
+  },
+];
+
+const { muiGridBlockContainer, muiGridBackground } = stylesBase;
+const useStyles = makeStyles({
+  muiGridBlockContainer,
+  muiGridBackground,
+  subtitle2: {
+    marginBottom: spacing(5),
+    fontWeight: 'bold',
+    width: '80%',
+  },
+  cardsContainer: {
+    margin: `0px 0px ${spacing(8)} -${spacing(2)}`,
+  },
+  cardContainer: {
+    width: '170px',
+    height: '170px',
+    margin: '5px',
+    boxShadow: '2px 2px 10px -4px black',
+    '&:hover #onHoverDisplay': { display: 'flex' },
+    '&:hover #onHoverHide': { display: 'none' },
+  },
+  cardOnHoverDisplay: {
+    height: '100%',
+    width: '100%',
+    display: 'none',
+    backgroundColor: colors.orange,
+  },
+  cardOnHoverHide: ({ backgroundColor }) => ({
+    height: '100%',
+    width: '100%',
+    backgroundColor,
+  }),
+  h4: {
+    width: '90%',
+    fontSize: fontSizing(2),
+    lineHeight: '18px',
+    textAlign: 'center',
+    marginTop: '14px',
+  },
+  imgComponent: { marginBottom: spacing(4) },
+  body2: {
+    fontSize: '13px',
+    lineHeight: 'normal',
+    textAlign: 'center',
+    padding: '6px',
+    margin: 'auto',
+  },
+  markStyle: {
+    width: 'auto',
+    left: '0px',
+    marginTop: '-350px',
+  },
+});
+
+const SDGBlock = () => {
+  const {
+    // eslint-disable-next-line no-shadow
+    muiGridBlockContainer,
+    // eslint-disable-next-line no-shadow
+    muiGridBackground,
+    subtitle2,
+    cardsContainer,
+    cardContainer,
+    cardOnHoverDisplay,
+    h4,
+    imgComponent,
+    body2,
+    markStyle,
+  } = useStyles();
+  return (
+    <Grid className={muiGridBlockContainer} container>
+      <img className={`${muiGridBackground} ${markStyle}`} src={markImage} alt="Our values" />
+      <Typography className={subtitle2} variant="subtitle2">
+        Engaging people through skill-based volunteering can take The UN Sustainable Development Goals impact to
+        the next level.
+      </Typography>
+      <Grid container className={cardsContainer}>
+        {sdgs.map(({ title, image, textOnOver, color }) => (
+          <Grid item className={cardContainer} key={title}>
+            <Grid
+              id="onHoverHide"
+              className={useStyles({ backgroundColor: color }).cardOnHoverHide}
+              container
+              direction="column"
+              alignItems="center"
+              justify="space-between"
+            >
+              <Typography className={h4} variant="h4" color="primary">{toFormattedText(title)}</Typography>
+              <img className={imgComponent} src={image} alt={title} />
+            </Grid>
+            <Grid id="onHoverDisplay" className={cardOnHoverDisplay}>
```

Comment:
This `id` is inside a `map` - it's not going to be unique, which although HTML allows it, is against the spec.

If you're using it for CSS targeting, make it a `class` instead.